index.js

```diff
@@ -81,6 +81,10 @@ class DiscordUtils {
     return !isNaN(number) && isFinite(number);
   }
 
+  static isObject (object) {
```

Comment:
ReferenceError: input is not defined
You made a typo, wrote `object` as parameter for the function but you actually perform on `input` which is not defined, you have to fix that.
`input` -> `object`